spec/rack/test/uploaded_file_spec.rb

```diff
@@ -41,7 +41,7 @@ def test_file_path
         require 'java'
         java_import 'java.lang.System'
 
-        20.times do |i|
+        50.times do |i|
```

Comment:
Why is this change here? It doesn't _really_ seem to have anything to do with the Rubocop parts to me. 😉